releasenotes/notes/apm-credit-cards-obfuscation-918935ff76b3d897.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    APM: Added credit card obfuscation. It is off by default and can be enabled using the
+    env. var. DD_APM_OBFUSCATION_CREDIT_CARDS_ENABLED or `apm_config.obfuscation.credit_cards.enabled`
```

Comment:
I kind of left it out on purpose, thinking it should not be used unless there are some sort of issues because it affects performance. Maybe I should add it 🤔